custom_cards/custom_card_haven_washer/custom_card_haven_washer.yaml

```diff
@@ -119,7 +120,7 @@ custom_card_haven_washer:
               && states[variables.ulm_custom_card_washer_power].state === "on") {
               if (!!variables.ulm_custom_card_washer_machine_state
                 && !!states[variables.ulm_custom_card_washer_machine_state]?.state
-                && ["run", "pause"].indexOf(states[variables.ulm_custom_card_washer_machine_state].state) >= 0) {
+                && String(states[variables.ulm_custom_card_washer_machine_state].state) !== variables.ulm_custom_card_washer_machine_stop_state) {
```

Comment:
`String()` is not needed as states from HA are always strings.
But it does no harm

Reply:
Thnx for the comment, I've removed the String() calls